Sprint-1/interpret/to-pounds.js

```diff
@@ -24,4 +24,9 @@ console.log(`£${pounds}.${pence}`);
 // Try and describe the purpose / rationale behind each step
 
 // To begin, we can start with
-// 1. const penceString = "399p": initialises a string variable with the value "399p"
+// 1. const penceString = "399p": Initialize a string variable with the value "399p"
+// 2. line 3-6 : function substring removes "p" from the end of the string and assign to new const
+// 3. line 8 : function check the length that is 3 or not if not then add "0" to the start of the string until length become 3
+// 4. line 9-12 : function substring remove 2 numbers in the end, it take all numbers except 2 last.
+// 5. line 14-16 : function substring take number last 2 numbers.
```

Comment:
Extra question: Would the code still work properly if we removed `.padEnd(2, "0")` from line 16?

Reply:
correct